background.js

```diff
@@ -1,7 +1,7 @@
-chrome.tabs.onUpdated.addListener(function(tabId, changeInfo, tab) {
+chrome.tabs.onUpdated.addListener(function(tabId) {
 	chrome.tabs.sendMessage(tabId, {
         message: 'update',
-	});
+	}, () => void chrome.runtime.lastError ); // Suppress error on Firefox
```

Comment:
What does `void` do?

Reply:
AFAIK, Firefox only prints the error to the console when it doesn’t find any code that handles the error so this is just a callback that sends the error to [void](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void) which evaluates it and then returns undefined.

Reply:
Ah, that is very cool 👍

Reply:
I think it would be cleaner to use [extraParameters](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/Tabs/onUpdated#addListener_syntax) to filter down to just the tabs where the content script is injected. I don't know how Chrome handles the extra parameter.

Reply:
Good idea, I made an issue: https://github.com/ajayyy/SponsorBlock/issues/155